share/spice/libraries/footer.handlebars

```diff
@@ -0,0 +1,6 @@
+{{#if latest_release}}
+<div>{{latest_release}} Latest Release</div>
+{{else}}
+<div>&nbsp;</div>
```

Comment:
some of the queries had no latest release version, so it's a holder for non-values